contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -583,9 +590,12 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         }
 
         // Otherwise, the deliverer address must be included in allowedRelayers.
-        for (uint256 i = 0; i < allowedRelayers.length; ++i) {
+        for (uint256 i; i < allowedRelayers.length;) {
             if (allowedRelayers[i] == delivererAddress) {
                 return true;
+                unchecked {
```

Comment:
I assume that this unchecked increment should be outside of the if block.

Reply:
@bernard-avalabs  You are right about it..Thanks!